emukit/bayesian_optimization/acquisitions/entropy_search.py

```diff
@@ -180,10 +184,11 @@ def evaluate(self, x: np.ndarray) -> np.ndarray:
         predicted_logP = np.subtract(predicted_logP, lselP)
 
         # We maximize the information gain
-        dHp = np.sum(np.multiply(np.exp(predicted_logP), np.add(predicted_logP, self.representer_points_log)), axis=0)
+        H_p = np.sum(np.multiply(np.exp(predicted_logP), np.add(predicted_logP, self.representer_points_log)), axis=0)
 
-        dH = np.mean(dHp)
-        return np.array([[dH]])
+        new_entropy = np.mean(H_p)
+        entropy_change = new_entropy - self.p_min_entropy
```

Comment:
I am not entirely sure but I think the averaging happens after we subtracted self.p_min_entropy, i e.:
`entropy_change = np.mean(H_p - self.p_min_entropy)`

Reply:
My understanding was that it's after. Equation (6) here:
http://jmlr.csail.mit.edu/papers/volume13/hennig12a/hennig12a.pdf

Either way, `self.p_min_entropy` is a scalar so the numerical result would be identical.